### DISCUSSION TITLE
Welcome to ITS_LIVE Xarray Tutorial Discussions!

### DISCUSSION THREAD

**Discussion by e-marshall** [e-marshall/itslive#32] [Announcements]
## 👋 Welcome!
Thanks for checking out our discussions page. We hope that you'll use this space to:
* Ask questions about tutorial code and content
* Engage with other users

If you have a question about a particular step or section in the tutorials, you can post it here. For bug reports and other feedback please check existing issues and raise an issue if appropriate.

**Comment by ZuhaFatima** [comment]
Hello everyone, I am wondering where can I get this parquet file.
se_asia = gpd.read_parquet('**rgi7_region15_south_asia_east.parquet**')
